master/reference/calicoctl/resources/policy.md

```diff
@@ -46,21 +46,23 @@ spec:
-| egress     | Ordered list of egress rules applied by this policy.                                                                                                                |                 | List of [Rule](#rule) |         |
-| doNotTrack | Indicates that the rules in this policy should be applied before any data plane connection tracking, and that packets allowed by these rules should not be tracked. | true, false     | boolean               | false   |
-
-The `doNotTrack` field is meaningful for [host
-endpoints]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/hostendpoint)
-only.  It does not apply at all to [workload
-endpoints]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/workloadendpoint);
-connection tracking is always used for flows to and from those.
-
-[Untracked policy]({{site.baseurl}}/{{page.version}}/getting-started/bare-metal/bare-metal) explains more about how `doNotTrack` can be useful for host endpoints.
+| Field      | Description                                                                                                                                           | Accepted Values | Schema                | Default |
+|------------|-------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------|-----------------------|---------|
+| order      | (Optional) Indicates priority of this policy, with lower order taking precedence.  No value indicates highest order (lowest precedence)               |                 | float                 |         |
+| selector   | Selects the endpoints to which this policy applies.                                                                                                   |                 | [selector](#selector) | all()   |
+| ingress    | Ordered list of ingress rules applied by policy.                                                                                                      |                 | List of [Rule](#rule) |         |
+| egress     | Ordered list of egress rules applied by this policy.                                                                                                  |                 | List of [Rule](#rule) |         |
+| doNotTrack | Indicates to apply the rules in this policy before any data plane connection tracking, and that packets allowed by these rules should not be tracked. | true, false     | boolean               | false   |
+| preDNAT    | Indicates to apply the rules in this policy before any DNAT.                                                                                          | true, false     | boolean               | false   |
+
+The `doNotTrack` and `preDNAT` fields are meaningful only when applying policy to a
+[host endpoint]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/hostendpoint).
```

Comment:
Yes, `doNotTrack` has so far been limited to host endpoints.  Also I'm pretty sure that the current _requirement_ for preDNAT only needs application to host endpoints.

I think it would be a doable but non-trivial addition to do preDNAT for workload endpoints too -
 so happy for considering this as a possible feature for 2.5.  (Think we need to
- make sure that relevant workload endpoint chains are in the mangle table
- program workload dispatch chains into the mangle table
- add rules into Calico's mangle PREROUTING chain to dispatch to the workload chains)